env.mjs

```diff
@@ -16,7 +16,8 @@ export const env = createEnv({
     NEXT_PUBLIC_PROFILE_ID: z.string().min(1),
     NEXT_PUBLIC_CONTEXT_ID: z.string().min(1),
     NEXT_PUBLIC_APP_URL: z.string().url(),
-
+    NEXT_PUBLIC_TACO_RITUAL_ID: z.string().min(1),
+    NEXT_PUBLIC_TACO_DOMAIN: z.string().min(1),
```

Comment:
Definitely very minor/unimportant, but just pointing out that this could potentially be:
```
import { domains } from `@nucypher/taco`;
...
    NEXT_PUBLIC_TACO_DOMAIN: z.enum([domains.DEVNET, domains.TESTNET, domains.MAINNET]),
```

if the import doesn't work then
```suggestion
    NEXT_PUBLIC_TACO_DOMAIN: z.enum(["lynx", "tapir", "mainnet"]),
```

Reply:
Good idea. Unfortunately I was not able to use the import, so I added the domains literally.

cbc58f855d9afb9e2d913a0553ba7aa55b2a451b